docs/parser.md

```diff
@@ -3,3 +3,94 @@ title: "Parser"
 weight: 60
 ---
 
+## Parser
+
+The AsyncAPI Parser is a package used to parse and validate the [AsyncAPI documents](asyncapi-document.md) in your Node.js or browser application. These documents can be either in YAML or JSON format.
+
+The Parser validates these documents using dedicated schema-supported plugins. 
+
+Supported schemas:
+
+- AsyncAPI schema (no plugin needed)
+- OpenAPI schema
+- JSON schema
+- Avro schema
+- RAML data-type schema
+
+The Parser allows the template developer to easily access schemas provided in the above supported formats. This is because the JavaScript parser converts all of them into JSON schema.
+
+If the document is valid, the Parser returns an `AsyncAPIDocument` instance with a set of helper functions that enable easier access to the contents of the AsyncAPI document. The parser provides dereferenced output. During the dereference process, the AsyncAPI parser substitutes a reference with a full definition. The dereferenced output is always in JSON format. The parser provides a message listing all errors if a document is invalid. The original asyncAPI document is part of the [Template Context](template-context.md) as the generator also passes the original asyncAPI document to the template context.	
+
+The following AsyncAPI document example has two channels: `channelOne` and `channelTwo`. Each channel has one operation and a single message:
+
+```yaml
+asyncapi: '2.5.0'
+info:
+  title: Demo API
+  version: '1.0.0'
+channels:
+  name/channelOne:
```

Comment:
In the paragraph above, you say there are two channels: `channelOne` and `channelTwo`. So let's remove `name/` from the name of the channel or it can cause confusion:

```suggestion
  channelOne:
```